docs: improve enrichment documentation (#1883)
aquasecurity/tracee: docs/index.md

    You may need to change the volume mounts for the kernel headers based on
    your setup. See [Linux Headers](./install/headers.md) section for more
    info.

> !!! note

Reviewer: The !!! stanza is for the mkdocs only (you have to render with make -f builder/Makefile.mkdocs and check through the web page).

>     Tracee supports enriching additional data from running containers into it's events.

Reviewer 2: ```suggestion
    Tracee supports enriching events with additional data from running containers.
```
I think this wording is more clear
